### PR TITLE
Fix inflation issue on lenovo

### LINKED ISSUE #9989: InflateException: Binary XML file line #2: Error inflating class <unknown>
Sentry Issue: https://sentry.io/share/issue/9aa4bcd3cb0c48faa34947b64b2d7f80/
```
RuntimeException: Failed to resolve attribute at index 3
    at org.wordpress.android.widgets.WPTextView.<init>(WPTextView.java:25)
...
(76 additional frame(s) were not displayed)

InvocationTargetException: None
    at java.lang.reflect.Constructor.newInstance(Constructor.java)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
    at android.view.LayoutInflater.createView(LayoutInflater.java:607)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:743)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:482)
...
(70 additional frame(s) were not displayed)

InflateException: Binary XML file line #2: Error inflating class <unknown>
    at org.wordpress.android.ui.FilteredRecyclerView$SpinnerAdapter.getView(FilteredRecyclerView.java:455)
...
(68 additional frame(s) were not displayed)

Binary XML file line #2: Error inflating class <unknown>
```

### PR DESCRIPTION
**Pull request by malinajirka**
Fixes #9989 and https://github.com/wordpress-mobile/WordPress-Android/issues/10168

It seems some styles are missing on Lenovo devices with Android 5.x. I don't have any such device, so I can't test the changes. But I'd give it a try.

To test:
1. Open reader
2. Make sure the currently selected filter looks ok 
3. Open the filter drop down and make sure it looks ok
-----
1. Open My site
2. Comments
3. Make sure the currently selected filter looks ok 
4. Open the filter drop down and make sure it looks ok

Update release notes:

- The changes are too minor

### CODE REVIEW

File: WordPress/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -85,18 +85,20 @@
...
 
-    <style name="FilteredRecyclerViewFilterTextView.WordPress" parent="android:TextAppearance.Widget.TextView">
+    <style name="FilteredRecyclerViewFilterTextView">
         <item name="android:layout_marginStart">0dp</item>
         <item name="android:paddingBottom">@dimen/margin_medium</item>
         <item name="android:paddingEnd">@dimen/margin_extra_large</item>
         <item name="android:paddingStart">10dp</item>
         <item name="android:paddingTop">@dimen/margin_medium</item>
         <item name="android:textSize">@dimen/text_sz_large</item>
+        <item name="textColor">@color/white</item>
```
Comment (theck13) [inline WordPress/src/main/res/values/styles.xml]:
```suggestion
        <item name="textColor">@android:color/white</item>
```

Can we change this to `@android:color/white`?  There is no resource `white` in `colors.xml` so using `@color/white` would be pulling from another source or library.

Comment (malinajirka) [reply WordPress/src/main/res/values/styles.xml]: ohh, good catch. I didn't realize I had enabled "suggestions from dependencies".

Comment (malinajirka) [reply WordPress/src/main/res/values/styles.xml]: Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10387/commits/be3c478d4198c852b8678ab7ff3b4f7faa633fef

**Comment by malinajirka** [comment]
Thanks @theck13 !!

>I don't have a Lenovo device so I can't reproduce the root cause or completely verify these changes fix the issue. Should we request another reviewer that has a Lenovo device to test?

Tbh I personally wouldn't waste time on it. We don't even know if it crashes on all Lenovo devices or just some specific models. It's not a major crash and these changes should be pretty safe. If the crash doesn't disappear, we can try to reproduce it.

**Pull request merged**